images/build

```diff
@@ -68,6 +68,9 @@ for IMAGE in "${!IMAGE_USERS[@]}"; do
     build_image ./image-user "$IMAGE" --build-arg "USER=$USER"
 done
 
+# Build the user with pre-defined group image
+build_image ./image-predefined-group test-image-predefined-group
+
```

Comment:
_continued from: https://github.com/kubernetes-sigs/cri-tools/pull/1005#discussion_r997929425_

hmm... this doesn't seem to push `gcr.io/k8s-staging-cri-tools/test-image-predefined-group` 

CI failed in 
```
failed to pull image: rpc error: code = Unknown desc = reading manifest latest in gcr.io/k8s-staging-cri-tools/test-image-predefined-group: manifest unknown: Failed to fetch "latest" from request "/v2/k8s-staging-cri-tools/test-image-predefined-group/manifests/latest".
```
ref: https://github.com/kubernetes-sigs/cri-tools/actions/runs/3270189869/jobs/5378524293#step:12:159

> We build images for each commit via Google Cloud Build: https://github.com/kubernetes-sigs/cri-tools/blob/master/images/build
> Adding the image there should work.

@saschagrunert Could you tell me how to see cloudbuild log?? I would like to debug why my image was not pushed.

Reply:
@everpeace the google cloud build job runs on `master` for each change of files with the `images` directory. I recommend to split-up the PR into two to add the image and later on use it.

Reply:
@saschagrunert oh, I see. I will do it. Thanks!